### PR TITLE
Escape search keyword

### PR DESCRIPTION
**Pull request by vkagamlyk**
Issue #, if available:

Description of changes:
Added escaping for search keywords. 
Searching for text containing quotes now works.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 **/node_modules/
 **/coverage/
 **/.DS_Store
+**/.idea/
+**/.vs/
```
Comment (alexey-temnikov) [inline .gitignore:5]:
```suggestion
**/.vs/

```

File: packages/graph-explorer/src/utils/escapeString.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export const escapeString = (text?: string): string => {
+  if (!text) {
+    return "";
+  }
+
+  console.log(text, text.includes('"') ? JSON.stringify(text).slice(1, -1) : text);
```
[the remark this answers is not in the document]
Comment (vkagamlyk) [reply packages/graph-explorer/src/utils/escapeString.ts]: right, removed

File: Changelog.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ This release includes the following feature enhancements and bug fixes:
 - Refactored API request options for non-IAM endpoints (<https://github.com/aws/graph-explorer/pull/230>)
 - Enforced JSON response format for SPARQL queries (<https://github.com/aws/graph-explorer/pull/230>)
 - Bumped `vite` to `4.5.2` (<https://github.com/aws/graph-explorer/pull/233>)
+- Searching for text containing quotes now works.
```
Comment (alexey-temnikov) [inline Changelog.md:14]: nit: Can we add an example (what didn't work before and what works now)?

**Review by alexey-temnikov — APPROVED** [review]

**Review by michaelnchin — APPROVED** [review]
Thank you @vkagamlyk @alexey-temnikov !

**Pull request merged**